### PR TITLE
Issue 84 column swap

### PR DESCRIPTION
**Pull request by JonasSchaub**
- swapped positions of sample parent structure and sample parent name columns in fragments table
- added text wrap to sample parent name column
- tweaked magic number for pref width in molecules tab a bit to get rid of the unnecessary scroll bars
- abbreviated some column header labels in the fragments tab and added tooltips with the un-abbreviated labels to those headers that did not have one yet

### CODE REVIEW

**Comment by JonasSchaub** [comment]
The exported PDF and CSV files did not have to be updated as described in the issue because they do not contain the sample parents or their names.

**Comment by JonasSchaub** [comment]
@FelixBaensch could you maybe inspect the visual changes in the GUI and let me know what you think?

File: src/main/resources/de/unijena/cheminf/mortar/message/Message_en_GB.properties [hunk 1]
```diff
@@ -92,16 +92,18 @@ MainTabPane.fragments.buttonPDF.txt = PDF
...
-MainTabPane.fragmentsTab.tableView.parentMolColumn.tooltip = First occurred molecule which contains fragment
-MainTabPane.fragmentsTab.tableView.parentMolNameColumn.header = Sample Parent
-MainTabPane.fragmentsTab.tableView.parentMolNameColumn.tooltip = First occurred molecule which contains fragment
+MainTabPane.fragmentsTab.tableView.moleculeFrequencyColumn.header = Mol. Frequency
+MainTabPane.fragmentsTab.tableView.moleculeFrequencyColumn.tooltip = Molecule Frequency
+MainTabPane.fragmentsTab.tableView.moleculePercentageColumn.header = Mol. Percentage
+MainTabPane.fragmentsTab.tableView.moleculePercentageColumn.tooltip = Molecule Percentage
+MainTabPane.fragmentsTab.tableView.parentMolColumn.header = Sample Parent Structure
+MainTabPane.fragmentsTab.tableView.parentMolColumn.tooltip = First fragmented molecule which contained the fragment
+MainTabPane.fragmentsTab.tableView.parentMolNameColumn.header = S. P. Name
```
Comment (FelixBaensch) [inline src/main/resources/de/unijena/cheminf/mortar/message/Message_en_GB.properties:105]: S. P. Name is somehow odd and I don't like it.

**Review by FelixBaensch — CHANGES_REQUESTED** [review]
I find the new arrangement takes a bit of getting used to. Maybe it looks better if the SMILES column is in front of the structure columns. 
And the heading "S. P. Name" is really odd.

**Comment by JonasSchaub** [comment]
> I find the new arrangement takes a bit of getting used to. Maybe it looks better if the SMILES column is in front of the structure columns.

Hm, I don't know. I think the most important thing here are the fragment structures. And textual column in front that is not a name/ID is also odd, IMHO.

image

**Comment by JonasSchaub** [comment]
> S. P. Name is somehow odd and I don't like it.

Do you have alternative suggestions, maybe? There is not a lot space available in the header and I don't like the header labels being abbreviated in the default size.

**Comment by FelixBaensch** [comment]
> Hm, I don't know. I think the most important thing here are the fragment structures. And textual column in front that is not a name/ID is also odd, IMHO.

I fully agree that the most important thing is the fragment structure, but it's more pleasing to my eye this way. We could get Achim's professional opinion.

**Comment by FelixBaensch** [comment]
> Do you have alternative suggestions, maybe? There is not a lot space available in the header and I don't like the header labels being abbreviated in the default size.

No, not really. How about just omitting the "Sample" and only mentioning in the tool tip and tutorial that there are several? And swap the "parent" for something like "source" or "root"?

**Comment by JonasSchaub** [comment]
> We could get Achim's professional opinion.

I've e-mailed him with you in Cc.

**Comment by FelixBaensch** [comment]
I like Achim's suggestion ;D

By the way, the arrangement of the columns can be customized by the user during a session. It is just not persisted.

**Comment by JonasSchaub** [comment]
> By the way, the arrangement of the columns can be customized by the user during a session. It is just not persisted.

Ah wow, I did not know that, thanks!

**Comment by JonasSchaub** [comment]
The new fragments tab arrangement:

image

@FelixBaensch do you like it?

**Comment by FelixBaensch** [comment]
> @FelixBaensch do you like it?

Since it pretty much implements my suggestions, yes!

**Review by FelixBaensch — APPROVED** [review]
Merge at will

**Pull request merged**